DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateCourses/DelegateCoursesViewModelTests.cs

```diff
@@ -0,0 +1,147 @@
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "G" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "H" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "I" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "J" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "K" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "L" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "M" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "N" },
+                new CourseStatisticsWithAdminFieldResponseCounts { ApplicationName = "O" },
+            };
+
+        [Test]
+        public void DelegateCoursesViewModel_should_default_to_returning_the_first_ten_delegates()
+        {
+            // When
+            var model = new DelegateCoursesViewModel(
+                courses,
+                new List<string>(),
+                new List<string>(),
+                null,
+                nameof(CourseStatistics.SearchableName),
+                GenericSortingHelper.Ascending,
+                null,
+                1,
+                null
+            );
+
+            // Then
+            using (new AssertionScope())
+            {
+                model.Courses.Count().Should().Be(BasePaginatedViewModel.DefaultItemsPerPage);
+                model.Courses.Any(c => c.CourseName == "K").Should()
+                    .BeFalse();
+            }
+        }
+
+        [Test]
+        public void DelegateCoursesViewModel_should_correctly_return_the_second_page_of_delegates()
+        {
+            // When
+            var model = new DelegateCoursesViewModel(
+                courses,
+                new List<string>(),
+                new List<string>(),
+                null,
+                nameof(CourseStatistics.SearchableName),
+                GenericSortingHelper.Ascending,
+                null,
+                2,
+                null
+            );
+
+            // Then
+            using (new AssertionScope())
+            {
+                model.Courses.Count().Should().Be(5);
+                model.Courses.First().CourseName.Should().BeEquivalentTo("K");
+            }
+        }
+
+        [Test]
+        public void DelegateCoursesViewModel_filters_should_be_set()
+        {
+            // Given
+            var categories = new[]
+            {
+                "Category 1",
+                "Category 2",
+            };
+            var topics = new[]
+            {
+                "Topic 1",
+                "Topic 2",
+            };
+
+            var expectedFilters = CourseStatisticsViewModelFilterOptions.GetFilterOptions(categories, topics);
```

Comment:
Nitpick, but I reckon this belongs in the `Then` section - that's generally where I'd expect to find expected values.